cmd/mt-kafka-mdm-sniff/main.go

```diff
@@ -61,6 +62,12 @@ func (ip inputPrinter) ProcessMetricData(metric *schema.MetricData, partition in
 	if *substr != "" && !strings.Contains(metric.Name, *substr) {
 		return
 	}
+	if *invalid {
+		err := metric.Validate()
+		if err == nil && metric.Time != 0 {
+			return
+		}
```

Comment:
minor optimization: `if metric.Time != 0 && metric.Validate() == nil` would save the execution of `Validate()` in the case where `Time == 0`